src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -11,200 +11,252 @@
-      verification: |
-
+      identity: "alpha"
+      command: "AddNOC"
+      cluster: "Operational Credentials"
+      arguments:
+          values:
+              - name: "NOCValue"
+                value: Node_Operational_Certificate_TH2
+              - name: "ICACValue"
+                value: Intermediate_Certificate_TH2
+              - name: "IPKValue"
+                value: IPK_TH2
+              - name: "CaseAdminSubject"
+                value: Commissioner_Node_Id_TH2
+              - name: "AdminVendorId"
+                value: 0xFFF1
+      response:
+          values:
+              - name: "StatusCode"
+                value: 0
+      # verification: "Verify that DUT responds with NOCResponse with status code OK"
+
+    - label: "Step 9: TH2 starts discovery of DUT using Operational Discovery"
+      # PICS: ""
+      # verification: ""
       disabled: true
 
-    - label:
-          "Step 11: TH2 opens a CASE session with DUT over operational network"
-      verification: |
-          DUT is able to open the CASE session with TH2
+    - label: "Step 10: TH2 opens a CASE session with DUT over operational network"
+      # PICS: ""
+      # verification: "DUT is able to open the CASE session with TH2"
       disabled: true
 
-    - label: "Step 12: TH2 sends CommissioningComplete command"
+    - label: "Step 11: TH2 sends CommissioningComplete command"
       PICS: CGEN.S.C05.Tx
-      verification: |
-          DUT respond with SUCCESS at CommissioningComplete command sent by TH2
-      disabled: true
-
-    - label:
-          "Step 13: TH1 does a non-fabric-filtered read of the Fabrics attribute
-          from the Node Operational Credentials cluster. Save the FabricIndex
-          for TH2s entry as TH2_Fabric_Index for future use."
+      nodeId: 0x43211234
+      identity: "beta"
+      cluster: "General Commissioning"
+      command: "CommissioningComplete"
+      response:
+          values:
+              - name: "ErrorCode"
+                value: 0 # SUCCESS
+      # verification: "DUT respond with SUCCESS at CommissioningComplete command sent by TH2"
+
+    - label: "Step 12.1: Save the FabricIndex for TH2's entry as TH2_Fabric_Index for future use."
       PICS: OPCREDS.S.A0001
-      verification: |
-          Verify that there are 2 entries in the list where one entry matches TH1_Fabric_Index and the other matches TH2_Fabric_Index.
-      disabled: true
-
-    - label:
-          "Step 14: TH2 does a non-fabric-filtered read of the Fabrics attribute
-          from the Node Operational Credentials cluster"
+      identity: "beta"
+      nodeId: 0x43211234
+      command: "readAttribute"
+      cluster: "Operational Credentials"
+      attribute: "CurrentFabricIndex"
+      response:
+          saveAs: TH2_Fabric_Index
+
+    - label: "Step 12.2: TH1 does a non-fabric-filtered read of the Fabrics attribute from the Node Operational Credentials cluster."
```

Comment:
This part looks identical to step 13...  This seems like a bug in the test plan: it should be reading CurrentFabricIndex, as this test does in step 12.1, not doing this weird "Save the FabricIndex for TH2’s entry" thing, when there is no way to tell what "TH2's entry" is.

Please file a test plan issue and just remove this part?

Reply:
@liamgonyea Yes, that would make sense to me.

Reply:
Test Plan Issue: https://github.com/CHIP-Specifications/chip-test-plans/issues/3396

Reply:
Addressed in commit 851d536a95d0d1232bca39f21086ba1ae215cc90